Bash_scripts/HW20/roles/hw202/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+#Get info from files
+- name: "Get variables from file"
+  import_tasks: template.yml
+
+#Copy to group linux
+- name: "Copy files to hosts"
+  become: true
+  become_user: root
+  ansible.builtin.copy:
+   src: "{{ item }}"
+   dest: "{{ dest_curl }}"
```

Comment:
What if files have different dests?

Reply:
they have different paths. Add variables to different hotspot groups. Each host group will have its own dest.path